lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -85,15 +85,21 @@ <h2>Byte efficiency tester page</h2>
 
   <!-- PASS(optimized): image is JPEG optimized -->
   <!-- FAIL(webp): image is not WebP optimized -->
-  <!-- FAIL(responsive): image is 25% used at DPR 2 -->
+  <!-- FAIL(responsive): image is 50% used at DPR 3 -->
   <!-- PASS(offscreen): image is onscreen -->
   <img style="width: 256px; height: 170px;" src="lighthouse-1024x680.jpg">
 
+  <!-- PASS(optimized): image is JPEG optimized -->
+  <!-- PASS(webp): image is WebP optimized -->
+  <!-- FAIL(responsive): image is % used at DPR 3 -->
```

Comment:
Should there be a percent here? Or like <XX% used? Where XX is the threshold?
```html
	  <!-- FAIL(responsive): image is XX% used at DPR 3 -->
```

Reply:
whoops, yes 25% used :)